packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -89,22 +83,19 @@ class TimelineScreenBodyState extends State<TimelineScreenBody>
     controller.timelineService.updateListeningState(true);
 
     cancel();
-    addAutoDisposeListener(controller.timelineModeNotifier);
-    addAutoDisposeListener(controller.fullTimeline.recordingNotifier, () {
+    addAutoDisposeListener(controller.recordingNotifier, () {
       setState(() {
-        recording = controller.fullTimeline.recordingNotifier.value;
+        recording = controller.recordingNotifier.value;
```

Comment:
nit: recordingNotifier seems like a bit of an awkward name when used here.
maybe it is just
controller.recording with a type ValueNotifier?

Reply:
updated this var and similar ones in TimelineController